lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -423,19 +428,34 @@ class SwiftASTContext : public TypeSystemSwift {
 
   CompilerType GetErrorType() override;
 
-  bool HasErrors();
+  /// Error handling
+  /// \{
+  bool HasErrors() const;
+  bool HasClangImporterErrors() const;
+
+  void RaiseDiagnostic(DiagnosticSeverity severity, llvm::StringRef message);
```

Comment:
"Raise" sounds like you're increasing the severity level. How about `ReportDiagnostic` and `ReportFatalError`. That would be consistent with the `Report*` methods in `Debugger` for example.